### PR TITLE
chore(Dockerfile*): standardize dockerfiles + add rhel7

### PR DESCRIPTION
**Pull request by ericavonb**

### CODE REVIEW

**Comment by ericavonb** [comment]
/assign @mrogers950
@openshift/sig-auth

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+WORKDIR /go/src/github.com/openshift/cluster-machine-appover
+COPY . .
+RUN make build
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
-COPY --from=builder /go/src/github.com/openshift/cluster-machine-appover/machine-approver /usr/bin/machine-approver
+COPY --from=builder /go/src/github.com/openshift/cluster-machine-appover/machine-approver /usr/bin/
 COPY manifests /manifests
-LABEL io.openshift.release.operator true
+ENTRYPOINT ["/usr/bin/cluster-machine-approver"]
```
Comment (mrogers950) [inline Dockerfile]: Should be /usr/bin/machine-approver

Comment (ericavonb) [reply Dockerfile]: 🤦‍♀️ fixed in cf16135

**Pull request merged**